index.bs

```diff
@@ -996,6 +1013,12 @@ spec:css-syntax-3;
             [=user mediation=], return [=a promise rejected with=]
             a "{{TypeError}}" {{DOMException}}.
 
+        1.  If |options|.{{CredentialRequestOptions/mediation}} is
```

Comment:
Note that as written, a request with both passwords and public key credentials would reject (since passwords seem to be implied to not support immediate given the text around the `immediate` definition).

Reply:
The earlier text was a bit ambiguous, since it says PublicKeyCredential has to be requested, but didn't exclude additional credential types. I've added text clarifying that Passwords can be requested along with WebAuthn creds (though not on their own, particularly since there is not yet any way to feature detect that).

Is that sufficient, or do you think something here needs to change as well?

Reply:
Probably good enough for now.